email/email.go

```diff
@@ -0,0 +1,50 @@
+package email
+
+import (
+	"log"
+	"net/mail"
+	"net/smtp"
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/scorredoira/email"
+)
+
+func SendEmail(emaill string, filename string) error {
+
+	err := godotenv.Load("./.env")
+	if err != nil {
+		log.Println("Couldn't load environment variables: ", err)
+		return err
+	}
+
+	from := os.Getenv("EMAIL")
```

Comment:
It's beneficial to write all names, such as EMAIL, in a helper or common file as constant variables and use them throughout a project to avoid mistakes.